### DISCUSSION TITLE
New User - 404 Error

### DISCUSSION THREAD

**Discussion by Wald99** [community/community#135901] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi
Sorry I am new to web publishing but I am getting a 404 error when I enter https://usrname.github.io/Repository/ into the browser
Background; I created a new Repository, uploaded files and folders and committed, although I did not enter any description, website, or topics at this stage
but now I get  a 404 error when I enter https://usrname.github.io/Repository/  into the browser.
The files and folders are on main
There is an index.html in main
The URL is using the correct case
I have cleared the cache
Github status is OK
I have 2 sub folders: css & images
The files referencing is relative and capitalisation is ok
The web page works locally (Win 10)
I am not using a custom domain

Any help would be greatly appreciated
Thank you

**Comment by davevad93** [comment]
Hi @Wald99 ,

Given the description you have provided I assume the repository you are talking about is [this](https://github.com/Wald99/Home), I've checked the website https://wald99.github.io/Home/ and it works fine, no 404 errors at all.

**Reply by davevad93** [reply, marked as answer]
Even though it's an old repo and you said it's new. [Home](https://github.com/Wald99/Home) is the only repository you have with a `css` and `images` folder with an active GitHub Pages deployment. [Lisa](https://github.com/Wald99/Lisa) and [GTest](https://github.com/Wald99/GTest) don't have any active deployments, so probably the issue is that you haven't built and deployed your pages, to do that go to your repo `Settings` => `Pages`.

![0000000](https://github.com/user-attachments/assets/91f636ef-6fb5-4f11-b2aa-ee38f6b78bcd)

**Reply by Wald99** [reply]
Thank you so much Dave. Much appreciated
I learnt html & css basics about 2 years ago, which is when my github was set up
By new I meant inexperienced. I knew I had to change the source to main in settings
but I had forgotten to go into pages to build and deploy
So Many Thanks
Geoff

**Reply by davevad93** [reply]
You're welcome @Wald99 , happy to help! 👍
